### PR TITLE
Malware cs fixes v2

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: N.a.

## Description
Fixing playbook description + adding additional conditional for the file detonation logic.

## Screenshots
N.A. 

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_9.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Updated playbook description.
+##### CrowdStrike Falcon - True Positive Incident Handling
+Updated playbook description.
+##### CrowdStrike Falcon - SIEM ingestion Get Incident Data
+Updated playbook description.
+##### CrowdStrike Falcon - False Positive Incident Handling
+Updated playbook description.
+##### CrowdStrike Falcon Malware - Investigation and Response
+- Updated playbook description.
+- Added additional filtering for file detonation logic.
```
Comment (altmannyarden) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_9.md:17]: let's change it to "Fixed file detonation logic" or "Added validation for file detonation logic"

**Review by altmannyarden — CHANGES_REQUESTED** [review]
just change the RN and approving

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_9.md (hunk 1 above)
Comment (ssokolovich) [reply Packs/CrowdStrikeFalcon/ReleaseNotes/1_8_9.md]: Done.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**